_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -54,49 +54,49 @@ There are no major accessibility support issues known for this rule.
 
 #### Passed Example 1
 
-[Visible][] label and [accessible name][] matches when trailing white spaces are removed.
+This link has [visible][] text that, ignoring trailing whitespace, matches the [accessible name][].
 
 ```html
-<div role="link" aria-label="next page ">next page</div>
+<a href="//w3.org" aria-label="W3C Website ">W3C Website</div>
 ```
 
 #### Passed Example 2
 
-Character insensitivity between [visible][] label and [accessible name][].
+This link has [visible][] text that, ignoring case, matches the [accessible name][].
 
 ```html
 <div role="link" aria-label="Next Page">next page</div>
 ```
 
 #### Passed Example 3
 
-Full [visible][] label is contained in the [accessible name][].
+This button has [visible][] text that is included in the [accessible name][].
 
 ```html
-<button name="link" aria-label="Next Page in the list">Next Page</button>
+<button aria-label="Next Page in the list">Next Page</button>
 ```
 
 ### Failed
 
 #### Failed Example 1
 
-[Visible][] label doesn't match [accessible name][].
+This link has [visible][] text that is different from the [accessible name][].
 
 ```html
 <div role="link" aria-label="OK">Next</div>
 ```
 
 #### Failed Example 2
 
-Not all of [visible][] label is included in [accessible name][].
+This button has [visible][] text that is only partially included in the [accessible name][].
 
 ```html
-<button name="link" aria-label="the full">The full label</button>
+<button aria-label="the full">The full label</button>
 ```
 
 #### Failed Example 3
 
-Mathematical symbols cannot be substituted for text as [explicitly mentioned in WCAG](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name#mathematical-expressions-and-formulae).
+This link has [visible][] text with mathematical symbols, this does not match the [accessible name][] because the mathematical symbols were written out in the accessible name. This is [explicitly mentioned in WCAG](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name#mathematical-expressions-and-formulae).
```

Comment:
```suggestion
This link has [visible][] text with mathematical symbols, that does not match the [accessible name][] because the mathematical symbols were written out in the accessible name. This is [explicitly mentioned in WCAG](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name#mathematical-expressions-and-formulae).
```